### PR TITLE
refactor: improve zk stack block explorer guide pt1

### PR DESCRIPTION
**Pull request by zk-Lumi**
# What :computer: 
* Remade guide for using block explorer
* Removed inaccurate wording for using block explorer
* Improved wording for using block explorer
* Split block explorer part into 2 sections -> 1) via docker 2) locally

# Why :hand:
* Part of the guide was inaccurate
* Also this section was quite confusing on how to use block explorer

### CODE REVIEW

**Comment by zk-Lumi** [comment]
Part 1 - This is to update the section on block explorer running locally, wording, and to remove inaccurate description.

Part 2- @githubdoramon will help update the section on block explorer running in docker in coming days.

File: docs/zk-stack/running-a-hyperchain/locally.md [hunk 1]
```diff
@@ -114,19 +114,59 @@ You can now navigate to the displayed Portal URL (typically <http://localhost:30
...
+Make sure you have your [zksync-era](https://github.com/matter-labs/zksync-era) repo set up locally and the `zk server` is running. The wizard in this guide allows you to run the server in the end. If you chose not to, you’re still able to run it by executing:
 
-Block explorer contains three components [Worker](https://github.com/matter-labs/block-explorer/tree/main/packages/worker), [API](https://github.com/matter-labs/block-explorer/tree/main/packages/api), and [App](https://github.com/matter-labs/block-explorer/tree/main/packages/app), which you can run all together locally and connect to your hyperchain. For that, you need to set up all the necessary environment and configuration files with your hyperchain settings. You can use a script to build them. See [setting up env variables](https://github.com/matter-labs/block-explorer#%EF%B8%8F-setting-up-env-variables).
+```bash
+zk server --components "http_api,eth,state_keeper,housekeeper"
+```
+
+Running the `zk server` builds environment and configuration files for block explorer based on your [zksync-era](https://github.com/matter-labs/zksync-era) repo configuration, which you can edit if you need any changes.
```
Comment (githubdoramon) [inline docs/zk-stack/running-a-hyperchain/locally.md:125]: This is not true - running it just runs it, there is nothing related to build config files. Just remove this sentence.

Or, say something like "The server is the main entry point to your hyperchain, and where the block explorer will fetch information to index from"

File: docs/zk-stack/running-a-hyperchain/locally.md [hunk 2]
```diff
@@ -114,19 +114,59 @@ You can now navigate to the displayed Portal URL (typically <http://localhost:30
...
+
+### Running block explorer in docker
```
Comment (githubdoramon) [inline docs/zk-stack/running-a-hyperchain/locally.md:127]: Based on the conversation we had in Slack, let's totally remove the docker option

File: docs/zk-stack/running-a-hyperchain/locally.md [hunk 3]
```diff
@@ -114,19 +114,59 @@ You can now navigate to the displayed Portal URL (typically <http://localhost:30
...
+Docker makes it easy to run the block explorer quickly. To run the block explorer in docker, please follow the readme for [Block Explorer](https://github.com/matter-labs/block-explorer).
 
-Once you have your [zksync-era](https://github.com/matter-labs/zksync-era) repo set up locally, you can run the following command to build environment and configuration files for block explorer based on your [zksync-era](https://github.com/matter-labs/zksync-era) repo configuration:
+If you need to modify the block explorer, you can run it locally instead.
+
+### Running block explorer locally
+
+#### Install block explorer
+
+Clone & install the block explorer:
```
Comment (githubdoramon) [inline docs/zk-stack/running-a-hyperchain/locally.md:137]: the block explorer repository

File: docs/zk-stack/running-a-hyperchain/locally.md [hunk 4]
```diff
@@ -114,19 +114,59 @@ You can now navigate to the displayed Portal URL (typically <http://localhost:30
...
-zk server --components "http_api,eth,state_keeper,housekeeper"
+cd /path/to/where/you/clone/repos
+git clone https://github.com/matter-labs/block-explorer.git
+cd block-explorer
+npm install
+```
+
+#### Setting up env variables
+
+Next you need to set up all the necessary environment and configuration files with your hyperchain settings. You can use a script to build them:
```
Comment (githubdoramon) [inline docs/zk-stack/running-a-hyperchain/locally.md:148]: You can use a script to set them up:

**Review by githubdoramon — CHANGES_REQUESTED** [review]
Requesting some small changed

**Review by githubdoramon — APPROVED** [review]

**Pull request merged**